setup.py

```diff
@@ -33,7 +33,10 @@
     ],
     extras_require={
         "metrics": "microcosm-metrics>=2.5.0",
-        "encryption": "aws-encryption-sdk>=2.0.0",
+        "encryption": [
+            "aws-encryption-sdk>=2.0.0",
+            "cryptography>=41",
```

Comment:
I had to bump `pyOpenSSL` to 23.2.0 due to dependency mismatch ( with 22~ it required cryptopgraphy to be < 40 for me)